docs/episode-01.md

```diff
@@ -0,0 +1,68 @@
+# Episode 1
+
+[Video](https://www.youtube.com/watch?v=wgHIkdUQbp0) |
```

Comment:
This link contains the exact start time: https://youtu.be/wgHIkdUQbp0?t=9m44s

Reply:
I thought about doing that but instead I've trimmed the video on YouTube. Unfortunately it seems to take them forever to process that :-(

Reply:
Great, that's better! Thanks for letting me know. I'll remove my "Starts at" comment on YouTube so as not to confuse folks. 😃